### PR TITLE
Corrected script for some item #5

### PR DESCRIPTION
**Pull request by idk-whoami**
* **Addressed Issue(s)**: 

* **Server Mode**: re

* **Description of Pull Request**: Implementations + corrections of a bunch of items

### CODE REVIEW

**Review by Lemongrass3110 — COMMENTED** [review]
Just some small comments i noticed while scrolling over it.

File: db/re/item_combos.yml [hunk 1]
```diff
@@ -24824,3 +24824,470 @@ Body:
     Script: |
       bonus bBaseAtk,20;
       bonus bMatk,20;
+  - Combos:
+      - Combo:
+          - aegis_490193     # 490193
+          - Hero     # 29509
+    Script: |
+      if (eaclass()&EAJL_THIRD && BaseJob == JOB_MINSTREL || eaclass()&EAJL_THIRD && BaseJob == JOB_WANDERER)
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_combos.yml]: Thanks!

File: db/re/item_combos.yml [hunk 2]
```diff
@@ -24824,3 +24824,470 @@ Body:
...
+          - Pendant_of_Solomon_     # 490036
+          - Amplification3     # 29447
+      - Combo:
+          - Pendant_of_Solomon_     # 490036
+          - Amplification4     # 29448
+      - Combo:
+          - Pendant_of_Solomon_     # 490036
+          - Amplification5     # 29449
+    Script: |
+      if (BaseClass == Job_Novice || BaseClass == JOB_Mage || BaseClass == Job_Acolyte || BaseClass == Job_Summoner) {
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_combos.yml]: Thanks!

File: db/re/item_combos.yml [hunk 3]
```diff
@@ -24824,3 +24824,470 @@ Body:
...
+          - Pendant_of_Solomon     # 490035
+          - Amplification3     # 29447
+      - Combo:
+          - Pendant_of_Solomon     # 490035
+          - Amplification4     # 29448
+      - Combo:
+          - Pendant_of_Solomon     # 490035
+          - Amplification5     # 29449
+    Script: |
+      if (BaseClass == Job_Novice || BaseClass == JOB_Mage || BaseClass == Job_Acolyte) {
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_combos.yml]: Thanks!

File: db/re/item_db_equip.yml [hunk 4]
```diff
@@ -6174,6 +6174,28 @@ Body:
          bonus2 bAddRace,RC_Undead,20;
          bonus2 bAddRace,RC_Angel,20;
       }
+  - Id: 1337
+    AegisName: Golden_Wrench_J
+    Name: Golden wrench    # !todo check english name
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_db_equip.yml:6179]: Thanks!

File: db/re/item_combos.yml [hunk 5]
```diff
@@ -25168,4 +25168,470 @@ Body:
...
+      bonus bMatk,100;
+      bonus2 bSubRace,RC_Player_Human,40;
+      bonus2 bSubRace,RC_Player_Doram,40;
+      skill "LK_CONCENTRATION",5;
+  - Combos:
+      - Combo:
+          - Arabian_Manteau     # 480023
+          - Ifrit_Card     # 4430
+    Script: |
+      bonus2 bSubEle,Ele_Fire,100;
```
Comment (Atemo) [inline db/re/item_combos.yml:25595]: The combo can stay like that but the description of some servers also reduce critical slash by 100% (30% on JRO)

File: db/re/item_combos.yml [hunk 6]
```diff
@@ -25168,4 +25168,470 @@ Body:
...
       bonus2 bSkillAtk,"GC_COUNTERSLASH",12*(.@r/2);
-
+  - Combos:
+      - Combo:
+          - aegis_490193     # 490193
+          - Hero     # 29509
+    Script: |
+      if ((eaclass()&EAJ_THIRDMASK)==EAJ_MINSTRELWANDERER)
+         bonus bNoSizeFix;
+         autobonus3 "{ bonus3 bAutoSpell,\"RA_UNLIMIT\",5,1000; }",10,20000,"WM_SEVERE_RAINSTORM","{}";
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_combos.yml]: Thanks!

File: db/re/item_db_equip.yml [hunk 7]
```diff
@@ -151306,6 +152330,32 @@ Body:
...
+      if (getskilllv("WM_UNLIMITED_HUMMING_VOICE") >= 5) {
+         bonus bHPGainValue,100;
+         bonus bSPGainValue,10;
+      }
+      if (getskilllv("WM_SEVERE_RAINSTORM") >= 5)
+         bonus2 bAddClass,Class_Boss,15;
+      if (getskilllv("WM_FRIGG_SONG") >= 5)
+         bonus bMaxHPrate,25;
+      if (getskilllv("WM_GLOOMYDAY") >= 5)
+         bonus bAtkRate,15;
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_db_equip.yml:152355]: Thanks!

File: db/re/item_combos.yml [hunk 8]
```diff
@@ -25168,4 +25168,470 @@ Body:
...
+  - Combos:
+      - Combo:
+          - Indecisive_P_Shoes     # 470078
+          - Hawkeye     # 4879
+      - Combo:
+          - Indecisive_P_Shoes_     # 470079
+          - Hawkeye     # 4879
+    Script: |
+      bonus bDex,15;
+      bonus bAtkRate,25;
```
[the remark this answers is not in the document]
Comment (idk-whoami) [reply db/re/item_combos.yml]: Thanks!

File: db/re/item_combos.yml (hunk 5 above)
Comment (idk-whoami) [reply db/re/item_combos.yml:25596]: Sealed Ifrit Card ( JRO Ver )
I will add card next PR

**Comment by Atemo** [comment]
Thanks!

**Pull request merged**